### PR TITLE
Added home page

### PR DESCRIPTION
**Pull request by srishti-maurya**
Added home page.

Note: Responsiveness would be added in the end

Live : https://deploy-preview-1--plan-it-shop-it.netlify.app/

screencapture-127-0-0-1-5500-2022-02-09-15_16_20

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+.home-display-category {
+  max-width: 300px;
+  color: var(--primary);
+  position: relative;
+  justify-self: center;
+}
+
+.home-display-category:hover {
+  box-shadow: rgba(0, 0, 0, 0.2) 0px 20px 30px;
```
Comment (msdeshmukh009) [inline styles/styles.css:30]: You might use this shadow again so better store it as a variable.

**Review by msdeshmukh009 — APPROVED** [review]
Looking good!!

**Review by someshtiwarii — APPROVED** [review]
Looks beautiful

File: styles/styles.css (hunk 1 above)
Comment (srishti-maurya) [reply styles/styles.css:30]: Thank you for the idea, I'll do that for sure

**Pull request merged**